pyanaconda/ui/gui/__init__.py

```diff
@@ -171,7 +171,7 @@ def _findUIFile(self):
             if os.path.isfile(testPath) and os.access(testPath, os.R_OK):
                 return testPath
 
-        raise IOError("Could not load UI file '%s' for object '%s'" % (self.uiFile, self))
+        raise Exception("Could not load UI file '{}' for object '{}'".format(self.uiFile, self))
```

Comment:
It is not recommended to raise the general exception. I think you can use `OSError` in this case.

Reply:
Thank you, done.